tests/schema_update_test.go

```diff
@@ -277,14 +289,132 @@ func TestVCSSchemaUpdate(t *testing.T) {
 		t.Fatalf("failed to create repository, error: %v", err)
 	}
 
+	// Provision an instance.
+	instanceDir := t.TempDir()
+	instance1Name := "testInstance1"
+	instance1Dir, err := ctl.provisionSQLiteInstance(instanceDir, instance1Name)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	environments, err := ctl.getEnvironments()
+	if err != nil {
+		t.Fatal(err)
+	}
+	var prodEnvironment *api.Environment
+	for _, environment := range environments {
+		if environment.Name == "Prod" {
+			prodEnvironment = environment
+			break
+		}
+	}
+	if prodEnvironment == nil {
+		t.Fatal("unable to find prod environment")
+	}
+
+	// Add an instance.
+	instance1, err := ctl.addInstance(api.InstanceCreate{
+		EnvironmentID: prodEnvironment.ID,
+		Name:          instance1Name,
+		Engine:        db.SQLite,
+		Host:          instance1Dir,
+	})
+	if err != nil {
+		t.Fatalf("failed to add instance, error: %v", err)
+	}
+	// Create an issue that creates a database.
+	databaseName := "testVCSSchemaUpdate"
+	createContext, err := json.Marshal(&api.CreateDatabaseContext{
+		InstanceID:   instance1.ID,
+		DatabaseName: databaseName,
+	})
+	if err != nil {
+		t.Fatal(fmt.Errorf("failed to construct database creation issue CreateContext payload, error: %w", err))
+	}
+	issue, err := ctl.createIssue(api.IssueCreate{
+		ProjectID:   project.ID,
+		Name:        fmt.Sprintf("create database %q", databaseName),
+		Type:        api.IssueDatabaseCreate,
+		Description: fmt.Sprintf("This creates a database %q.", databaseName),
+		// Assign to self.
+		AssigneeID:    project.Creator.ID,
+		CreateContext: string(createContext),
+	})
+	if err != nil {
+		t.Fatalf("failed to create database creation issue, error: %v", err)
+	}
+	status, err := ctl.waitIssuePipeline(issue.ID)
+	if err != nil {
+		t.Fatalf("failed to wait for issue %v pipeline %v, error: %v", issue.ID, issue.Pipeline.ID, err)
+	}
+	if status != api.TaskDone {
+		t.Fatalf("issue %v pipeline %v is expected to finish with status done, got %v", issue.ID, issue.Pipeline.ID, status)
+	}
+	issue, err = ctl.patchIssueStatus(api.IssueStatusPatch{
+		ID:     issue.ID,
+		Status: api.IssueDone,
+	})
+	if err != nil {
+		t.Fatalf("failed to patch issue status %v to done, error: %v", issue.ID, err)
+	}
+
 	// Simulate Git commits.
+	file1 := "bbtest/Prod/testVCSSchemaUpdate__ver1__migrate__create_a_test_table.sql"
 	pushEvent := &gitlab.WebhookPushEvent{
 		ObjectKind: gitlab.WebhookPush,
+		Ref:        "refs/heads/master",
```

Comment:
nit: use a branch like feature/foo instead of master (we once have a bug https://github.com/bytebase/bytebase/issues/396)

Reply:
Done

Reply:
should be refs/heads/feature/foo

Reply:
Done.